dynamo/tools/velocyto_scvelo.py

```diff
@@ -212,6 +213,135 @@ def colormap_fun(x: np.ndarray) -> np.ndarray:
     return data_out
 
 
+def scv_dyn_convertor(adata: anndata, mode: Literal["to_dyn", "tp_scv"] = "to_dyn"):
+    """Convert the adata object used in Scvelo to the adata object used by Dynamo, or vice versa.
+
+    Conversion may need manual adjustments based on the use case. For example, all Dynamo anndata objects will be set
+    to the conventional experiment with the deterministic model. The `pp` information needs modification to enable other
+    methods.
+
+    Args:
+        adata: the adata object to be converted.
+        mode: the string indicates the mode. Mode `to_dyn` will convert Scvelo anndata object to Dynamo anndata object.
+            mode `to_scv` will convert Dynamo anndata to Scvelo anndata.
+
+    Returns:
+        The adata object after conversion.
+    """
+    main_info("Dynamo and scvelo have different preprocessing procedures and velocity estimation methods. "
+              "The conversion of adata may not be optimal for every use case, requiring potential manual adjustments.")
+    if mode == "to_dyn":
+        main_info("Start converting Scvelo adata into Dynamo adata...")
+        main_info("Scvelo data wil be converted into Dynamo adata with the conventional assumption and the"
+                  "deterministic model. If this is not what you want, please change them manually.")
+        if "highly_variable_genes" in adata.var.columns:
+            adata.var["pass_basic_filter"] = adata.var.pop("highly_variable_genes")
+            adata.var["pass_basic_filter"] = [True if item == 'True' else False for item in
+                                              adata.var["pass_basic_filter"]]
```

Comment:
I think they should use `use_for_pca` instead of pass_basic_filter

Reply:
The Scvelo preprocessing will create a var called `.var["highly_variable"]`, which will be converted to the `use_for_pca`. This `highly_variable_genes` is something from the example dataset itself. If we don't need this conversion, we can delete it.